app/controllers/admin/pages_controller.rb

```diff
@@ -0,0 +1,69 @@
+class Admin::PagesController < ApplicationController
+  before_action :require_admin
+  before_action :set_webbook, only: [:index, :show, :new, :edit, :update, :destroy]
+  before_action :set_page, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @pages = @webbook.pages.rank(:row_order)
+  end
+
+  def show
+  end
+
+  def new
+    @page = Page.new
+  end
+
+  def create
+    @page = Page.new(page_params.merge(webbook_id: params[:webbook_id], page_number: Webbook.find_by(id: params[:webbook_id]).pages.last.page_number + 1))
+    if @page.save
+      flash[:success] = '新しいページを作成しました'
+      redirect_to admin_webbook_pages_path
```

Comment:
success と notice 使いわけているとかでなければいかで良さそう

```suggestion
      redirect_to admin_webbook_pages_path, notice: '新しいページを作成しました'
```